iozone/client.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+# enable some more output
+set -x
+
+[ -n "${SERVER}" ]
+[ -n "${EXPORT}" ]
+
+# install build and runtime dependencies
+yum -y install nfs-utils time centos-release-gluster
+
+mkdir -p /mnt/ganesha
+
+#cd ~
+#curl -o iozone3_394.tar http://www.iozone.org/src/current/iozone3_394.tar
+
+#tar xvf iozone3_394.tar 
+#cd iozone3_394/src/current
+#make
+#make linux
+#ioZoneDir=$(pwd)/iozone
```

Comment:
Could you remove these leftovers, please? There are some more further below, the #ioZoneDir commands.

Reply:
@nixpanic Sure .. :)

Reply:
@nixpanic All comments are removed from the script ... :)